### ISSUE TITLE
Add ContentType for Webwarp & WebAlign

### ISSUE THREAD

**Issue by UlrikeS91** [openMetadataInitiative/openMINDS_instances#51]
This is one half of the original issue #10.

Both Webwarp and WebAlign have been registered and published on the Knowledge Graph:
Webwarp (0.7): https://search.kg.ebrains.eu/instances/06358a83-5bf0-4271-b482-1efc10208a0e 
WebAlign (v0.7): https://search.kg.ebrains.eu/instances/5258054a-8755-428d-8d0a-cc7a344e95d9

We should register these content types now. @majpuc suggested 

```
{
  "@context": {
    "@vocab": "https://openminds.ebrains.eu/vocab/"
  },
  "@id": "https://openminds.ebrains.eu/instances/contentTypes/application/vnd.webalign.waln",
  "@type": "https://openminds.ebrains.eu/core/ContentType",
  "fileExtension": [
    ".waln"
  ],
  "name": "application/vnd.webalign.waln",
  "relatedMediaType": null,
  "synonym": [
    "WebAlign waln file"
  ]
}
```
with comment: "The waln file contains image registration information. This file type was created for WebAlign and don't exist elsewhere."

and
```
{
  "@context": {
    "@vocab": "https://openminds.ebrains.eu/vocab/"
  },
  "@id": "https://openminds.ebrains.eu/instances/contentTypes/application/vnd.webwarp.wwrp",
  "@type": "https://openminds.ebrains.eu/core/ContentType",
  "fileExtension": [
    ".wwrp"
  ],
  "name": "application/vnd.webwarp.wwrp",
  "relatedMediaType": null,
  "synonym": [
    "WebWarp wwrp file"
  ]
}
```
with comment: "The wwrp file contains image registration information. This file type was created for WebWarp and don't exist elsewhere."

@lzehl Do you think these make sense? Or should parts be changed? If these look fine, I can make the PR.

PS: I actively ignored NutilWeb since it is not published on the KG yet. @majpuc, please open a new issue once the content types are actually needed.

**Comment by Majpuc** [comment]
Yes, I think you can create these two content type. Could we just ask @Tevemadar for a final check?

**Comment by lzehl** [comment]
@Majpuc @UlrikeS91 I would add a "description" and modify the "synonyms" of these content types to reflect that you are talking not about one file but a software specific file format (content type). The description should potentially include the base file format (e.g. json, ascii text, csv, etc) and more details on the content (inlc structure if helpful).

From the current description (comments) they both contain the same type of data and it is unclear what the difference is. (that they are specific for the software is clear since it would be registered as the software specific content type; which is deducible already from the name)

**Comment by Tevemadar** [comment]
Hi,
Yes, they're simple abbreviations.
There's no written specification for these formats yet.
They're closely related, technically `wwrp` files contain a single optional field per section on top of a `waln` file, that field encodes nonlinear deformation.

**Comment by lzehl** [comment]
@Tevemadar could you please provide us with a concrete suggestion. I've started writing something for the description for each content type below but please provide a correction / extension. How these dictionaries are actually structured (what keys, what values) is hard coded in the software, correct? If that is the case, then you can provide a link to the code as specification of these formats. If not, the description is sufficient write now.

WebAlign (content type filename: `application_vnd.webalign.waln.jsonld`):

```
{
  "@context": {
    "@vocab": "https://openminds.ebrains.eu/vocab/"
  },
  "@id": "https://openminds.ebrains.eu/instances/contentTypes/application_vnd.webalign.waln",
  "@type": "https://openminds.ebrains.eu/core/ContentType",
  "description": "WebAlign waln format is used for text-based files containing the registration information for multiple tissue section images to ??? in form of a dictionary. ",
  "fileExtension": [
    ".waln"
  ],
  "name": "application/vnd.webalign.waln",
  "relatedMediaType": null,
  "synonym": [
    "WebAlign waln format"
  ]
}
```

Webwarp (content type filename: `application_vnd.webwarp.wwrp.jsonld`):
```

{
  "@context": {
    "@vocab": "https://openminds.ebrains.eu/vocab/"
  },
  "@id": "https://openminds.ebrains.eu/instances/contentTypes/application_vnd.webwarp.wwrp",
  "@type": "https://openminds.ebrains.eu/core/ContentType",
  "description": "WebWarp wwrp format is used for text-based files containing the same registration information for multiple tissue section images to ??? in form of a dictionary as the WebAlign waln format. However this format has the option to additionally add information on the nonlinear deformation of each tissue section image.",
  "fileExtension": [
    ".wwrp"
  ],
  "name": "application/vnd.webwarp.wwrp",
  "relatedMediaType": null,
  "synonym": [
    "WebWarp wwrp format"
  ]
}
```

**Comment by lzehl** [comment]
@Majpuc and @Tevemadar we need your feedback / help to complete these content types. Could you please have a look at the descriptions and provide corrections/completions?

**Comment by Tevemadar** [comment]
I'll try to look at them on Friday.

**Comment by Tevemadar** [comment]
While I don't really know what that field means, they may be a relatedMediaType to https://github.com/openMetadataInitiative/openMINDS_instances/blob/main/instances/latest/contentTypes/application_vnd.ebrains.image-service.deepzoom.jsonld

> WebAlign waln format is used for text-based files containing the registration information for multiple tissue section images to a standard atlas space in form of a dictionary. Internally it is a JSON file that contains complete metadata for a collection of deepzoom images (mandatory), their location in the EBRAINS storage infrastructure (mandatory), identifier of the standard atlas space (mandatory), and the actual spatial registration in the form of 3D vector triplets per image (optional).

> WebWarp wwrp format is used for text-based files containing the same registration information for multiple tissue section images to a standard atlas space in form of a dictionary as the WebAlign waln format. However this format has the option to additionally add information on the nonlinear deformation of each tissue section image. Internally it is the same JSON as WebAlign waln, extended with a list of 2D deformation vectors per image (optional).

I don't know the level of detail needed here, the descriptions can be stripped of the "Internally" parts, to get a simple one.

**Comment by lzehl** [comment]
@Tevemadar thank you for the provision of the description and sorry that I did not follow up on this for such a long time. I have a question for the contentType descriptions: you are mentioning EBRAINS here. Is this format/contentType actually only working for EBRAINS? or could this be some other storage infrastructure?

For the relatedMediaTypes: this would refer to matching entries in the IANA.org registry (which you do not have I assume for your formats/contentTypes).

**Comment by Tevemadar** [comment]
Hi @lzehl, these changes are okay.
